### DISCUSSION TITLE
Switch from YAPF to Ruff

### DISCUSSION THREAD

**Discussion by JonathanPlasse** [nutti/fake-bpy-module#166] [Ideas]
By running [Scalene](https://github.com/plasma-umass/scalene) on `gen.py`. I found out that the formatting take 92% of the time on my computer.
I would suggest using Ruff instead for a much faster formatting.

**Comment by nutti** [comment]
@JonathanPlasse 

Thanks for your suggestion.
I agree with you about the slow formatting by using YAPF.
You can change the formatting tool and make a pull request.

**Comment by nutti** [comment]
Maybe, this discussion can be closed now.
@JonathanPlasse Thank you for tackling this topic.
